src/main/scala/org/constellation/domain/p2p/PeerHealthCheck.scala

```diff
@@ -49,9 +56,12 @@ class PeerHealthCheck[F[_]](cluster: Cluster[F], apiClient: ClientInterpreter[F]
     for {
       _ <- logger.info(s"Verifying node responsiveness: ${id}")
       peer <- cluster.getPeerInfo.map(_.get(id))
+      clock <- getClock
       result <- peer
         .map(_.peerMetadata.toPeerClientMetadata)
-        .fold(PeerUnresponsive.asInstanceOf[PeerHealthCheckStatus].pure[F])(checkPeer(_, confirmationCheckTimeout))
+        .fold(PeerUnresponsive(clock, 1).asInstanceOf[PeerHealthCheckStatus].pure[F])(
```

Comment:
I am wondering if `widen` can be used there instead of casting 🤔 
Or if it even makes a difference

Reply:
hm,, no difference imo. especially that widen is on the wrapper, not on the value itself